src/tools/compiletest/src/main.rs

```diff
@@ -489,15 +489,26 @@ fn stamp(config: &Config, testpaths: &TestPaths) -> PathBuf {
 }
 
 fn up_to_date(config: &Config, testpaths: &TestPaths, props: &EarlyProps) -> bool {
+    let rust_src_dir = config.find_rust_src_root().expect(
+        "Could not find Rust source root",
+    );
     let stamp = mtime(&stamp(config, testpaths));
-    let mut inputs = vec![
-        mtime(&testpaths.file),
-        mtime(&config.rustc_path),
-    ];
+    let mut inputs = vec![mtime(&testpaths.file), mtime(&config.rustc_path)];
     for aux in props.aux.iter() {
-        inputs.push(mtime(&testpaths.file.parent().unwrap()
-                                         .join("auxiliary")
-                                         .join(aux)));
+        inputs.push(mtime(
+            &testpaths.file.parent().unwrap().join("auxiliary").join(
+                aux,
+            ),
+        ));
+    }
+    // Relevant pretty printer files
+    let pretty_printer_files = [
```

Comment:
For this part, I don't know if the more relevant way to do is to "crawl" the `rust_src_path`, and to check if file which contains in the name "pretty_print". Using this solution, if a member creates a new pretty printer file, this file will be automatically checked in `up_to_date`...
The cons of this automated solution is that we must respect title rules for the pretty printing files.

Reply:
The explicit list is fine for now.

Reply:
But could you please add `lldb_batchmode.py` and `lldb_rust_formatters.py` as well?

Reply:
No problem!